hydra-node/test/Hydra/Model.hs

```diff
@@ -485,7 +485,8 @@ instance
         performAbort party
       Wait delay ->
         lift $ threadDelay delay
-      ObserveConfirmedTx tx -> do
+      ObserveConfirmedTx var -> do
+        let tx = ctx var
```

Comment:
Could: rename `ctx` to `lookupVar` as it is would be more clear to me that it corresponds to this `Lookup m` function from `quickcheck-dynamic`

Reply:
how about `lookup` only? this would lead to the code saying:
```
let tx = lookup var
...
```
which seems more pleasing to the eye

Reply:
If it does not conflict with any imports (`lookup` is quite generic)